src/intTest/resources/application.yml

```diff
@@ -130,6 +163,6 @@ nhais:
       SO01=fake_mailboxId
       UNY5=fake_mailboxId}
     scanMailboxIntervalInMilliseconds: ${NHAIS_SCAN_MAILBOX_INTERVAL_IN_MILLISECONDS:1000}
-    scanMailboxDelayInSeconds: ${NHAIS_SCAN_MAILBOX_DELAY_IN_SECONDS:1}
+    scanMailboxDelayInSeconds: ${NHAIS_SCAN_MAILBOX_DELAY_IN_SECONDS:2}
```

Comment:
Does this make our tests run more slowly?

Reply:
This is already on develop - I had to update it to 2 as RECEPs for inbound UAT were downloaded from MESH before I could turn off scheduler